migrations/2_alter_project_table.js

```diff
@@ -0,0 +1,18 @@
+exports.up = function (knex, Promise) {
+  return Promise.join(
+    knex.schema.table('projects', function (t) {
+      t.boolean('readonly');
+      t.text('client');
+    })
+  );
+};
+
+exports.down = function (knex, Promise) {
+  // Note that this is irreversible, and will lead to data loss.
+  return Promise.join(
+    knex.schema.table('projects', function (t) {
+      t.dropColumn('readonly');
```

Comment:
I wonder if we should also have this do some reversible, like rename the column or something...